sympy/core/function.py

```diff
@@ -551,6 +551,19 @@ def bad(m):
             if any(bad(a) for a in args):
                 raise ValueError  # one or more args failed to compute with significance
         except ValueError:
+            from sympy.functions.elementary.exponential import exp
```

Comment:
Core modules should avoid importing from outside core. Would this work for functions in general?

Reply:
Okk @jksuom I will avoid importing module from outside core.